content/development/clients/clients.en.yaml

```diff
@@ -1,19 +1,51 @@
-title: Clients
-description: Download links for various Ethereum Classic clients and deployment tools
-contribute: true
-disclaimer: true
-intro: | 
-  # Clients
-
-  Clients are used to sync with the Ethereum Classic network and verify blocks. Most users won't need to run a full node, but they allow those who do to trustlessly interact with the network. Ethereum Classic is scheduled for a network update in June, 2020. Here are [Ethereum Classic Client Upgrade Instructions](https://hackmd.io/IPA7zPwpQDy-1U0reeppcQ).
-releases: Releases
-support: Support
-guide: Guide
-publicEndpointsTitle: | 
-  ## Public Endpoints
-
-  You can use these services to connect to ETC via JSON-RPC.
-deploymentTools: | 
-  ## Node Deployment Tools
-
-  These tools make it easy to spin up a node or cluster of nodes for production deployments.
+type: info
+buttons: 
```

Comment:
All of the main content is now dynamically generated from the `content` folder structure.

The `type` in these files defines what component is used...